packages/jest-circus/README.md

```diff
@@ -0,0 +1,40 @@
+<h1 align="center">
+  <img src="https://jestjs.io/img/jest.png" height="150" width="150"/>
+  <img src="https://jestjs.io/img/circus.png" height="150" width="150"/>
+  <p align="center">jest-circus</p>
+  <p align="center">The next-gen test runner for Jest</p>
+</h1>
+
+## Overview
+
+Circus is a flux-based test runner for Jest that is fast, easy to maintain, and simple to extend.
```

Comment:
might be nice to clarify "flux-based"? 

Overall this is super clear and helpful, though, thanks!

Reply:
Should we link somewhere like https://github.com/facebook/flux/tree/master/examples/flux-concepts?

Reply:
just linking to https://github.com/facebook/flux is probably enough, I just assumed it was some different "flux", since I associate flux with UI (makes sense in retrospect though).

Reply:
fwiw it's based on the flux philosophy, it doesn't use the flux library specifically